alerting/src/main/kotlin/org/opensearch/alerting/MonitorMetadataService.kt

```diff
@@ -185,10 +187,16 @@ object MonitorMetadataService :
 
     suspend fun recreateRunContext(metadata: MonitorMetadata, monitor: Monitor): MonitorMetadata {
         try {
-            val monitorIndex = if (monitor.monitorType == Monitor.MonitorType.DOC_LEVEL_MONITOR)
+            val monitorIndex = if (
+                monitor.isMonitorOfStandardType() &&
```

Comment:
why not just check string value is doc level monitor?

Reply:
just want to use a single string constant, so just reusing the enum value.